src/network/networkprotocol.h

```diff
@@ -589,27 +599,31 @@ enum ToClientCommand
 		u32 id
 	*/
 
-	TOCLIENT_NUM_MSG_TYPES = 0x54,
+	TOCLIENT_SRP_BYTES_S_B = 0x60,
+	/*
+		Belonging to AUTH_MECHANISM_LEGACY_PASSWORD and AUTH_MECHANISM_SRP.
+
+		u16 command
+		std::string bytes_s
+		std::string bytes_B
+	*/
+
+	TOCLIENT_NUM_MSG_TYPES = 0x61,
 };
 
 enum ToServerCommand
 {
-	TOSERVER_INIT = 0x02,
+	TOSERVER_INIT = 0x0f,
```

Comment:
In fact, no new _INIT packet must be created later, it's the goal with 
this new _INIT packet, when the protocol v25 will be done. The _INIT 
packet shouldn't be changed for years